src/burnchains/burnchain.rs

```diff
@@ -111,6 +111,7 @@ impl BurnchainStateTransition {
         sort_tx: &mut SortitionHandleTx,
         parent_snapshot: &BlockSnapshot,
         block_ops: &Vec<BlockstackOperationType>,
+        missed_commits: &Vec<MissedBlockCommit>,
```

Comment:
These are the `MissedBlockCommits` that showed up in _this_ block, right?  i.e. they were destined for some earlier block up to `BURN_BLOCK_MINED_AT_MODULUS` blocks ago, but they were bumped here?

Reply:
Yep